src/resources/nfts/simplehash/index.ts

```diff
@@ -41,8 +42,9 @@ export async function fetchSimpleHashNFTs(
   cursor: string = START_CURSOR
 ): Promise<{ data: SimpleHashNFT[]; nextCursor: string | null }> {
   const chainsParam = RainbowNetworks.filter(network => network.features.nfts)
-    .map(network => network.nfts?.simplehashNetwork || network.value)
+    .map(network => network.nfts.simplehashNetwork || network.value)
```

Comment:
dont need the fallback anymore yeah? the null type may end up causing issues here tho  🤔

Reply:
trueeee